src/main/java/com/spotify/logging/LoggingConfigurator.java

```diff
@@ -433,7 +435,7 @@ public static void configure(final File file, final String defaultIdent) {
 
   // TODO (bjorn): We probably want to move this to the utilities project.
   // Also, the portability of this function is not guaranteed.
-  private static String getMyPid() {
+  public static String getMyPid() {
```

Comment:
Is it worth copying the code to avoid exposing it (since it isn't guaranteed to be portable and doesn't really belong here)?